demos/dotnet/dotnet/OtherActions.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Text;
+
+namespace dotnet
+{
+    internal static class OtherActions
```

Comment:
This seems like another dumping-ground class. This class represents the ProjectAgent and the BuildAgent... no?